test/models/event_test.rb

```diff
@@ -0,0 +1,97 @@
+class EventTest < ActiveSupport::TestCase
+
+  test "should not save event without title" do
+    event = Event.new
+    assert_not event.save
+  end
+
+  test "event timeline json should be correct" do
+    event = FactoryBot.create(:event)
+    expected_json = [
+      {
+        year: 2017,
+        past_year: true,
+        months: [
+          {
+            month: "jan",
+            past_month: true,
+            events: [
+              {
+                title: "Awesome event",
+                category: "Intergovernmental processes",
+                start_year: 2017,
+                start_month: "01",
+                start_day: "01",
+                end_year: 2018,
+                end_month: "01",
+                end_day: "01",
+                current_event: false,
+                past_event: true,
+                location: "London, UK",
+                organisers: [],
+                summary: "This is awesome",
+                relevance: nil,
+                outputs: "http://www.unep-wcmc.org",
+                cbd_relation: nil
```

Comment:
I think your branch is not up to date with master as here there should also be the `is_provisional_date` attribute. You can pull master and then merge it into your branch and make the required change.
Apart from this, it looks good!

Reply:
Ah I see thanks @Levia! I will sort it out now!